ThunderCloud/StormFile.swift

```diff
@@ -0,0 +1,57 @@
+//
```

Comment:
Can we make this `public` scope, as it'll come in handy in other projects/frameworks!!